### ISSUE TITLE
"title" and "placeholder" not consistently supported for name calculation

### ISSUE THREAD

**Issue by jasonkiss** [w3c/html-aam#84]
_From @patrickhlauke on March 30, 2017 22:44_

while on windows (testing briefly with jaws and nvda and various browsers - chrome, firefox, IE11, edge) it seems that `placeholder` and `title` are indeed used (when nothing else is present) to provide a name for a control (e.g. an `input`), this is not currently the case everywhere. just doing a quick test on mobile:

- on iOS/Safari/VoiceOver, once the user enters anything into an `input` named only with a `placeholder`, the placeholder text is not announced anymore (so you just get "input - whatever the user typed"; `title` does work (even when the user entered something, it is announced)
- on Android/Chrome/TalkBack, in both cases, as soon as the user entered something into the `input`, neither `placeholder` nor `title` are announced anymore

not tested on OS X, but I'll hazard a guess it behaves the same as iOS/VO

question now is: are these browsers wrong and should bugs be filed against them? or should `placeholder` and `title` be removed from the name calculation here? http://rawgit.com/w3c/html-api-map/master/index.html#accessible-name-and-description-calculation /cc @stevefaulkner 

_Copied from original issue: w3c/html-api-map#5_

**Comment by patrickhlauke** [comment]
super-basic test file i used 
[placeholder-title.zip](https://github.com/w3c/html-aam/files/884155/placeholder-title.zip)

**Comment by jasonkiss** [comment]
Snap. Was just creating my own. Good timing.

Can confirm that with VO in Safari in MacOS Sierra, both `placeholder` and `title` **are** announced as the name for the field, even after other content is entered by the user, as per the HTML-AAM.

See related discussion on `placeholder` as backup accname for `input` in #41.

Regardless, the ship seems to have sailed, and if `placeholder` is to be used this way, I'd consider what you found in iOS/Safari/VO and Android/Chrome/TalkBack to be bugs.

What say you @stevefaulkner?

**Comment by patrickhlauke** [comment]
so filing the issue with `placeholder` in iOS/Safari/VO and Android/Chrome/TalkBack, as well as the same issue with `title` in Android/Chrome/TalkBack (not actually checked Android/Firefox, should probably do that too), as bugs in those browsers, with reference to HTML-AAM? i don't mind doing that, just double-checking...

**Comment by jasonkiss** [comment]
Sounds right to me. 

In attempt to save from filing needless bugs, pinging @cookiecrook: any chance for a quick steer? 

Don't know who to ask re Android/Chrome/TalkBack.

**Comment by patrickhlauke** [comment]
> Don't know who to ask re Android/Chrome/TalkBack

i'll tap my Google sources for that :)

**Comment by cookiecrook** [comment]
https://webkit.org/b/170334

**Comment by cookiecrook** [comment]
@alice @minorninth

**Comment by patrickhlauke** [comment]
Filed https://bugs.chromium.org/p/chromium/issues/detail?id=707194

**Comment by jasonkiss** [comment]
Thanks, all. Closing as relevant bugs filed with browsers.
